<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modifier_joueur</class>
 <widget class="QDialog" name="Modifier_joueur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Veuillez selectionner l'id de joueur a modifier d'apes le tableau suivant:</string>
   </property>
  </widget>
  <widget class="QTableView" name="afficher_modif">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_modif">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="in_date">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>poids:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_nom">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_prenom">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_image">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>434</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>464</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="in_taille">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>466</y>
     <width>64</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>404</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Id:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_id">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>405</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Taille:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="in_poids">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>436</y>
     <width>64</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="id_sup_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>191</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="in_number">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>445</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Appliquer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <zorder>label_modif</zorder>
  <zorder>label</zorder>
  <zorder>afficher_modif</zorder>
  <zorder>in_date</zorder>
  <zorder>label_4</zorder>
  <zorder>in_nom</zorder>
  <zorder>in_prenom</zorder>
  <zorder>b_image</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>in_taille</zorder>
  <zorder>label_6</zorder>
  <zorder>in_id</zorder>
  <zorder>label_5</zorder>
  <zorder>label_7</zorder>
  <zorder>in_poids</zorder>
  <zorder>id_sup_8</zorder>
  <zorder>in_number</zorder>
  <zorder>label_8</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
